continuous_integration/circle_ci_test_doc.sh

```diff
@@ -0,0 +1,10 @@
+#!bin/bash
+
+# on circle ci, each command run with it's own execution context so we have to
+# activate the conda testenv on a per command basis. That's why we put calls to
+# python (conda) in a dedicated bash script and we activate the conda testenv
+# here.
+source activate testenv
+
+# pipefail is necessary to propagate exit codes
+set -o pipefail && cd doc && make html-strict 2>&1 | tee ~/log.txt
```

Comment:
FYI the reason warnings are treated as errors in nilearn is that before we were trying to get rid of warnings on each release which was nobody's definition of fun. Most of the warnings come from docstrings and are very very obscure. By having to fix them on each PR we were hoping that it would be easier to maintain.